### DISCUSSION TITLE
flashlight pause abuse fix?

### DISCUSSION THREAD

**Discussion by Axilotl17** [ppy/osu#18247] [Ideas] [answered]
disclaimer: I have no idea the solution but thought id bring it up
someone I know is trying to FC maps with FL by pause spamming in stable, and looking at the map in Lazer. this obviously isn't something legitimate...

**Comment by smoogipoo** [comment, marked as answer]
There's already a pause cooldown in lazer, though it's only 1s long.

**Reply by Axilotl17** [reply]
thats... not what i was saying...

**Reply by Axilotl17** [reply]
im talking about using lazer, to pause abuse in stable.

**Reply by smoogipoo** [reply]
Oh. There's nothing that can be actioned on then. The same can be said for any program or website that can read and display .osu files.

**Reply by Axilotl17** [reply]
:(
